extras/README.md

```diff
@@ -1,16 +1,102 @@
-# New Actor Template
-Use cargo generate to create a new actor module from this template.
+## How to use  built-in `wasmcloud:extras` capability provider
 
-[Learn how to write](https://wascc.dev/tutorials/first-actor/) waSCC actors, or [explore the actor concept](https://wascc.dev/docs/concepts/actors/).
+`wasmcloud` host runtime comes with some built-in capability providers and `wasmcloud:extras` is one of them. In this tutorial we will generate a random number and retrieve some information like request ID and number of requests using `wasmcloud:extras`. 
```

Comment:
actually we only have one built-in provider that's available to actors. We could change this first sentence to:

The wasmCloud host runtime comes with a built-in capability provider (`wasmcloud:extras`) that includes some useful and fairly common tools.